File: src/main/java/kr/co/lunatalk/LunatalkServerApplication.java
package kr.co.lunatalk;

import org.springframework.boot.SpringApplication;
import org.springframework.boot.autoconfigure.SpringBootApplication;
import org.springframework.scheduling.annotation.EnableScheduling;

@SpringBootApplication
@EnableScheduling
public class LunatalkServerApplication {

	public static void main(String[] args) {
		SpringApplication.run(LunatalkServerApplication.class, args);
	}

}

File: src/main/kotlin/kr/co/lunatalk/LunatalkServerApplication.kt
package kr.co.lunatalk

import org.springframework.boot.autoconfigure.SpringBootApplication
import org.springframework.boot.runApplication
import org.springframework.scheduling.annotation.EnableScheduling

@SpringBootApplication
@EnableScheduling
open class LunatalkServerApplication

fun main(args: Array<String>) {
    runApplication<LunatalkServerApplication>(*args)
}
